app/config/settings.py

```diff
@@ -140,6 +140,7 @@ def strtobool(val) -> bool:
 JQFILEUPLOAD_UPLOAD_SUBIDRECTORY = "jqfileupload"
 IMAGE_FILES_SUBDIRECTORY = "images"
 EVALUATION_FILES_SUBDIRECTORY = "evaluation"
+COMPONENTS_FILES_SUBDIRECTORY = "components"
```

Comment:
Is this used?

Reply:
Yes, it is, my search doesn't work and this should have been part of a review not a single comment...